gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Nov 18 15:38:00 PST 2020
```

Comment:
unneeded.  please remove.  it makes assessing the history of the file difficult.

Reply:
Sorry, oversight. Removed.